src/main/java/org/tron/common/net/udp/handler/MessageHandler.java

```diff
@@ -26,45 +26,45 @@
 import java.net.InetSocketAddress;
 import java.util.function.Consumer;
 import org.slf4j.LoggerFactory;
+import org.tron.common.overlay.discover.node.NodeManager;
 
 
-public class MessageHandler extends SimpleChannelInboundHandler<DiscoveryEvent>
-    implements Consumer<DiscoveryEvent> {
+public class MessageHandler extends SimpleChannelInboundHandler<UdpEvent>
+    implements Consumer<UdpEvent> {
 
   static final org.slf4j.Logger logger = LoggerFactory.getLogger("MessageHandler");
 
   public Channel channel;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make channel a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=951&issues=AWPTJmx0A0k93xG_IQRx&open=AWPTJmx0A0k93xG_IQRx)

[ISSUE_KEY]: AWPTJmx0A0k93xG_IQRx